cki_lib/Makefile

```diff
@@ -0,0 +1,9 @@
+run:
+	echo nothing > /dev/null
+
+METADATA=testinfo.desc
+
+$(METADATA): Makefile
+	@touch $(METADATA)
```

Comment:
Why `touch` if the next line has `> $(METADATA)`?

Reply:
Good question! It was just copied/pasted from other Makefiles in the tree. This pattern is in pretty much every Makefile for Beaker tasks and has been around for many years.

Reply:
@emanchado just an fyi, next phase to remove restraint-rhts is to convert all the Makefile's to use a metadata file instead. 
https://restraint.readthedocs.io/en/latest/tasks.html#metadata